src/types/MovieTypes.ts

```diff
@@ -0,0 +1,7 @@
+export interface MovieTypes {
```

Comment:
avoid `type, interface, enum` words in the variables names

```suggestion
export interface Movie {
```